backend/query_conversion.go

```diff
@@ -0,0 +1,32 @@
+package backend
+
+import (
+	"context"
+)
+
+// QueryConversionHandler is an EXPERIMENTAL service that allows converting queries between versions
```

Comment:
Based on this should it live in an experimental package?

Reply:
could be but I am just following the same approach than the other new handlers.